app/code/Magento/Backend/Block/Store/Switcher.php

```diff
@@ -616,4 +616,17 @@ public function isUsingIframe()
         }
         return false;
     }
+
+    /**
+     * Get whether confirm popup is being used
+     *
+     * @return bool
+     */
+    public function getUseConfirm()
+    {
+        if ($this->hasData('use_confirm')) {
+            return (bool)$this->getData('use_confirm');
+        }
+        return false;
+    }
```

Comment:
This class is marked as API, so we can't add new methods due to the backward compatibility policy.